### ISSUE TITLE
lan2san needs to account for board position

### ISSUE THREAD

**Issue by cshancock** [rosawojciech/bigchess#8]
Hello - 

Thank you for your package. I am using the lan2san function but it quickly occurred to me that it does not account for the current board state. For example, a move in lan of "e4d5" could either be a pawn on e4 making a capture, or a bishop on e4 moving diagonally one square (with or without a capture). 

Another example is f8c8. This could be a rook move or a queen move. 

Currently, passing "f8c8" to lan2san returns 1. Bxc8 -- of course, it is impossible for the bishop to move from the f8 square to the c8 square.

**Comment by rosawojciech** [comment]
Yes, in the current version the lan2san() uses the initial state of the chessboard and does not check if the move is correct
